care/facility/api/viewsets/facility.py

```diff
@@ -39,6 +41,14 @@ class FacilityFilter(filters.FilterSet):
     state = filters.NumberFilter(field_name="state__id")
     state_name = filters.CharFilter(field_name="state__name", lookup_expr="icontains")
     kasp_empanelled = filters.BooleanFilter(field_name="kasp_empanelled")
+    exclude_user = filters.BooleanFilter(method='filter_exclude_user')
+
+    def filter_exclude_user(self, queryset, name, value):
+        if value:
+            user = self.request.query_params.get('exclude_user')
+            if user:
+                queryset = queryset.exclude(facilityuser__user__username=user)
```

Comment:
```suggestion
            queryset = queryset.exclude(facilityuser__user__username=value)
```
value is the vlaue you are looking for